pkg/cloud/cloud_test.go

```diff
@@ -575,23 +575,6 @@ func TestCreateDisk(t *testing.T) {
 			expCreateVolumeInput: nil,
 			expErr:               fmt.Errorf("invalid StorageClass parameters; specify either IOPS or IOPSPerGb, not both"),
 		},
-		{
-			name:       "fail: io1 with too low iopsPerGB",
-			volumeName: "vol-test-name",
-			diskOptions: &DiskOptions{
-				CapacityBytes: util.GiBToBytes(4),
-				Tags:          map[string]string{VolumeNameTagKey: "vol-test", AwsEbsDriverTagKey: "true"},
-				VolumeType:    VolumeTypeIO1,
-				IOPSPerGB:     1,
-			},
-			expDisk: &Disk{
-				VolumeID:         "vol-test",
-				CapacityGiB:      4,
-				AvailabilityZone: defaultZone,
-			},
-			expCreateVolumeInput: nil,
-			expErr:               fmt.Errorf("invalid IOPS: 4 is too low, it must be at least 100"),
-		},
```

Comment:
Let's update the test case to reflect the new expected behavior - io1 with too low iopsPerGB should still fail server-side:

```
{
			name:       "fail: io1 with too low iopsPerGB",
			volumeName: "vol-test-name",
			diskOptions: &DiskOptions{
				CapacityBytes: util.GiBToBytes(4),
				Tags:          map[string]string{VolumeNameTagKey: "vol-test", AwsEbsDriverTagKey: "true"},
				VolumeType:    VolumeTypeIO1,
				IOPSPerGB:     1,
			},
			expDisk: &Disk{
				VolumeID:         "vol-test",
				CapacityGiB:      4,
				AvailabilityZone: defaultZone,
			},
			expCreateVolumeInput: &ec2.CreateVolumeInput{
				Iops: aws.Int64(4),
			},
			expErr:             fmt.Errorf("could not create volume in EC2: InvalidParameterValue: An error occurred (InvalidParameterValue) when calling the CreateVolume operation\ncaused by: Volume iops of 4 is too low; minimum is 100."),
			expCreateVolumeErr: awserr.New("InvalidParameterValue", "An error occurred (InvalidParameterValue) when calling the CreateVolume operation", fmt.Errorf("Volume iops of 4 is too low; minimum is 100.")),
		},
```

Reply:
Wouldn't that test the exact same thing as this existing test? https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/190f526d8290b2e7686e9432251a18feea3a34ac/pkg/cloud/cloud_test.go#L461-L471

Reply:
Yes, that explains why code coverage remains unchanged - resolving.